src/averageAges.js

```diff
@@ -14,12 +14,25 @@
  *
  * @return {number}
  */
+
+const filterByGender = (gender, people) => {
+  return people.filter(({ sex }) => sex === gender);
```

Comment:
here you could just make variable - each element of people (lets say person) and since its an object check the value of the key 'sex'